packages/app/components/checkout/select-plan.tsx

```diff
@@ -0,0 +1,255 @@
+          <Skeleton height={16} width={200} colorMode={colorMode as any} />
+        ) : defaultPaymentMethod ? (
+          <View tw="flex-row items-center">
+            <Checkbox
+              checked={selectDefault}
+              onChange={() => setSelectDefault(!selectDefault)}
+              accesibilityLabel="Select default payment method"
+            />
+            <Text
+              tw="ml-2 text-gray-900 dark:text-gray-50"
+              onPress={() => setSelectDefault(!selectDefault)}
+            >
+              Use my default payment method.
+            </Text>
+            <Pressable
+              tw="flex-row items-center pl-2"
+              onPress={() => setSelectDefault(!selectDefault)}
+            >
+              <CreditCard
+                width={20}
+                height={20}
+                color={isDark ? colors.white : colors.black}
+              />
+              <Text tw="ml-1 text-gray-900 dark:text-gray-50">
+                Ending in {defaultPaymentMethod.details.last4}
+              </Text>
+            </Pressable>
+          </View>
+        ) : null}
+      </View>
+      <View tw="mb-4 mt-8 items-center p-4">
+        <Text tw="text-lg font-semibold text-gray-900 dark:text-gray-50">
+          Why is it paid?
+        </Text>
+        <Text tw="pt-4 text-gray-900 dark:text-gray-50">
+          Polygon NFTs cost money to mint, so we made it as affordable as
+          possible.
+          {`\n\n`}
+          This way, your fans don’t need a wallet: you can give away free
+          collectibles to people who are new to web3!
+        </Text>
+      </View>
+      <Button
+        size="regular"
+        onPress={handleSelectPlan}
+        disabled={isMutating || paymentStatus === "processing"}
+        tw={
+          isMutating || paymentStatus === "processing" ? "opacity-[0.45]" : ""
+        }
+      >
+        <Text tw="font-semibold text-gray-50 dark:text-gray-900">
+          {isMutating || paymentStatus === "processing"
+            ? "Loading..."
+            : `Drop ${selectedPlan?.edition_size.toLocaleString("en-US", {
+                maximumFractionDigits: 2,
+              })} editions`}
+        </Text>
+      </Button>
+      {error ? (
+        <Text tw="pt-2 text-red-500">{error?.message}. Please retry</Text>
+      ) : null}
+      {paymentStatus === "failed" || paymentStatus === "notSure" ? (
+        <Text tw="pt-2 text-red-500">{paymentStatusMessage}</Text>
+      ) : null}
+    </View>
+  );
+};
+
+async function fetchPaymentIntent(
+  _url: string,
+  { arg }: { arg: { useDefaultPaymentMethod: boolean; dropPlan: DropPlan } }
+) {
+  if (arg) {
+    try {
+      const res = await axios({
+        method: "POST",
+        url: "/v1/payments/start",
+        data: {
+          payment_plan: arg.dropPlan.name,
+          use_default_payment_method: arg.useDefaultPaymentMethod,
+        },
+      });
+
+      return res;
+    } catch (e: any) {
```

Comment:
e is type of AxiosError

Reply:
Done! thanks for catching, i sometimes do this out of laziness. Will take care next time 🥺